src/parsers/manifest/dash/parse_mpd.ts

```diff
@@ -320,11 +320,17 @@ function parseCompleteIntermediateRepresentation(
     }
   }
 
+  const isTerminated = !isDynamic &&
```

Comment:
I would say that `isTerminated` is always true for `static` manifest now?

To me, for `dynamic` MPDs it is only true when:
  1. The duration is set (either through `mediaPresentationDuration` or through the end of the last Period
  2. the `minimumUpdatePeriod` attribute is not set.
 
Or at least, that's how I interpret DASH-IF v4.3 chapter 4.6.4

Thoughts?